app/feature/feature-changeaddress/src/main/kotlin/com/hedvig/android/feature/changeaddress/data/MoveQuote.kt

```diff
@@ -34,22 +35,25 @@ data class MoveQuote(
           postalCode = "124$index",
           street = "Froedingsvaegen $index",
         ),
-        numberCoInsured = index,
+        numberInsured = index,
         premium = UiMoney(99.0 * index, CurrencyCode.SEK),
         startDate = LocalDate(2023, 5, 13),
         isExpanded = index == 1,
         productVariant = ProductVariant(
           displayName = "Test",
-          typeOfContract = "testTpe",
+          contractType = ContractType.RENTAL,
           partner = "test",
-          product = Product(
-            displayNameFull = "Test",
-            pillowImageUrl = "",
-          ),
           perils = emptyList(),
-          insurableLimits = emptyList(),
+          insurableLimits = listOf(
+            InsurableLimit(
+              label = "test",
+              description = "longlonglonglonglonglonglonglonglonglonglonglonglonglonglonglong",
```

Comment:
A thing I like to do in these scenarios is
```suggestion
              description = "long".repeat(10),
```
It's very convenient